src/pages/purchase-orders/edit/hooks/useHandleLineItemPropertyChange.ts

```diff
@@ -26,12 +26,20 @@ export function useHandleLineItemPropertyChange(
     value: unknown,
     index: number
   ) => {
+    const updatedPurchaseOrderLineItems = purchaseOrder.line_items;
+
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    updatedPurchaseOrderLineItems[index][property] = value;
+
```

Comment:
Oof, can you explain why we do this?

Reply:
The logic has been changed, so this is removed!